app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -305,7 +305,12 @@ def handle_expired_status_update(enrollment, response, response_message)
     end
 
     begin
-      send_deadline_passed_email(enrollment.user, enrollment) unless enrollment.deadline_passed_sent
+      unless enrollment.deadline_passed_sent
+        send_deadline_passed_email(
+          enrollment.user, enrollment,
+          'none'
+        )
```

Comment:
Nit: I'd suggest either all arguments on one line, or one argument per line. I find mixed single-vs-multiple arguments per line to be difficult to parse. It also seems like this could all fit on one line within the line limit?

```suggestion
        send_deadline_passed_email(enrollment.user, enrollment, 'none')
```